jenkins/test/testsuite/Jenkinsfile

```diff
@@ -62,26 +59,60 @@ pipeline {
             steps {
                 script {
                     basePath = "https://ci.opensearch.org/ci/dbc/bundle-build/${opensearch_version}/${build_id}/${platform}/${architecture}"
+                    sh "wget ${basePath}/builds/manifest.yml"
                     sh "./test.sh ${JOB_NAME} ${basePath} --test-run-id ${env.BUILD_NUMBER}"
                 }
             }
             post() {
                 always {
                     script {
-                        publish()
+                        uploadTestResults(
+                            upload: true,
+                            jobName: "bundle_build",
+                            buildNumber: "${build_id}"
+                        )
                     }
                     cleanWs disableDeferredWipeout: true, deleteDirs: true
                 }
             }
         }
     }
-}
+    post() {
+        success {
+            node('Jenkins-Agent-al2-x64-m5xlarge') {
+                script {
+                    def stashed = lib.jenkins.Messages.new(this).get(['integ-test'])
+                    publishNotification(
+                        icon: ':white_check_mark:',
+                        message: 'Successful Build',
```

Comment:
Should this be `Integration tests successful`? Also below `Failed integration tests`?